test/test_auditable.py

```diff
@@ -0,0 +1,296 @@
+# -*- coding=utf8 -*-
+from __future__ import unicode_literals
+import unittest
+
+from rdflib import Graph, Namespace
+from rdflib.plugins.stores.auditable import AuditableStore
+
+EX = Namespace("http://example.org/")
+
+
+class TestAuditableStore(unittest.TestCase):
+
+    def setUp(self):
+        self.g = Graph()
+        self.g.add((EX.s0, EX.p0, EX.o0))
+        self.g.add((EX.s0, EX.p0, EX.o0bis))
+
+        self.t = Graph(AuditableStore(self.g.store),
+                       self.g.identifier)
+
+    def test_add_commit(self):
+        self.t.add((EX.s1, EX.p1, EX.o1))
+        self.assertItemsEqual(self.t, [
```

Comment:
py3 tests fail cause it seems `assertItemsEqual` is deprecated since 3.2

maybe just rewrite as nose test?